agent/Equalizer.py

```diff
@@ -255,21 +283,28 @@ def get_highest_safe_card(self,
         suit_to_choose_from.sort()
         # get the cards already played in this trick
         trick_cards = self.played_cards_in_trick(partial_state)
+        hearts_played = 0;
```

Comment:
`;` old habits die hard =). Python will just ignore it.

Reply:
:/ Pycharm was quite happy to not inform me of that semicolon.